src/main/java/graphql/language/AbstractNode.java

```diff
@@ -14,4 +18,12 @@ public void setSourceLocation(SourceLocation sourceLocation) {
     public SourceLocation getSourceLocation() {
         return sourceLocation;
     }
+
+    public List<String> getComments() {
+        return comments;
+    }
+
+    public void setComments(List<String> comments) {
```

Comment:
I would prefer to assert that comments is not null and throw an exception if it is.